sms/dlr-flask.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask, request, jsonify
+from pprint import pprint
+app = Flask(__name__)
+
+
+@app.route("/receipt", methods=['GET', 'POST'])
+def delivery_receipt():
+    if request.method == 'POST':
+        pprint(request.json)
```

Comment:
Only just discovered that `request.json` [is deprecated](http://flask.pocoo.org/docs/0.12/api/#flask.Request.json)